File: Programação/Back-End PHP/07/ExemploOO/Correção exercicio 1/Pessoa.php
<?php
    class Pessoa {
        // Atributo
        private string $nome;
        
        // Comportamentos
        public function getNome() : string {
            return $this->nome;
        }

        public function setNome(string $nome) {
            $this->nome = $nome;
        }
    }
?>

File: Outros/POO UML/Aula 10/br.com.uninove.empresa/src/br/com/uninove/empresa/Pessoa.java
package br.com.uninove.empresa;

public class Pessoa {
	
	private String nome;
	
	public Pessoa(String nome)
	{
		this.nome = nome;
	}
	
	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}
	
	public void mostraClasse()
	{
		System.out.println("Classe Pessoa");
	}
	
}
